src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, useState } from 'react';
+import { NavbarBrand } from './NavbarBrand';
+import { NavbarCollapse } from './NavbarCollapse';
+import { NavbarContext } from './NavbarContext';
+import { NavbarToggle } from './NavbarToggle';
+
+export type NavbarCompnentProps = {
+  open?: boolean;
```

Comment:
I think we can rename this to `sidebarOpen` or something similar since it should be related to the state of the sidebar.